Userland/Applications/Browser/Tab.cpp

```diff
@@ -407,10 +407,12 @@ Optional<URL> Tab::url_from_location_bar(MayAppendTLD may_append_tld)
     String text = m_location_box->text();
 
     StringBuilder builder;
-    builder.append(text);
+
+    String textUrl = g_search_engine.replace("{}", text);
+    builder.append(textUrl);
+
     if (may_append_tld == MayAppendTLD::Yes) {
-        // FIXME: Expand the list of top level domains.
```

Comment:
Has this actually been fixed?

Reply:
just enhance some condition, but I'm not fixed that above, I only enhance for if someone types a keyword on the URL tab, then the URL tab also automatically redirects into the search engine, depending on what search engine used

Reply:
Please don't remove a FIXME if you didn't fix the issue.

Reply:
I already restored that fix me comment :^)